src/ContactList.js

```diff
@@ -20,45 +14,43 @@ class ContactList extends Component {
                 avatar={contact.avatar}
                 occupation={contact.occupation}
                 active={contact.active}
-                onChange={this.AddContact.bind(this)}
+                onButtonClick={() => this.props.onContactClick(contact._id)}
               />
             );
           })}
         </ul>
-        <div>
-          <h2>Added Contacts</h2>
-          <div>
-            {this.state.AddedContacts.map(name => {
-              return (
-                <div className="added-item" key={name}>
-                  <p>{name}</p>
-                </div>
-              );
-            })}
-          </div>
-        </div>
       </div>
 
     );
   }
 
-  AddContact(name) {
-    const Added = this.state.AddedContacts;
-    if (Added.includes('no contacts')) {
-      Added[0] = name;
-    } else if (!Added.includes(name)) {
-      Added.push(name);
-    }
-    this.setState({
-      AddedContacts: Added
-    });
-  }
+  // reset() {
```

Comment:
I had the same naming for my reset button and Ivo asked me to put it into a clickHandler and handle the execution elsewhere